src/mca/pdl/pdlopen/pdl_pdlopen_module.c

```diff
@@ -175,7 +175,26 @@ static int pdlopen_foreachfile(const char *search_path,
 
         dp = opendir(dirs[i]);
         if (NULL == dp) {
-            ret = PMIX_ERR_IN_ERRNO;
+            if (0 == i) {
```

Comment:
Hmmm...only thing I don't like is that C:B didn't give you a reasonable error message. I would never go from an unknown np option to guessing that my component path was wrong. I guess the reason my error message didn't get triggered is that at least one directory existed - but no components could be found.

Problem is that some frameworks are okay without components, so I don't know how we can generalize an error message. I'm guessing mpirun came up with that error because the ORTE schizo framework found no usable components, and thus the cmd line was never defined. I'll see if I can at least deal with that one.